api/nakadi-event-bus-api.yaml

```diff
@@ -2328,6 +2386,62 @@ definitions:
         description: |
           Number of milliseconds that Nakadi stores events published to this event type.
 
+  EventTypeAuthorization:
+    type: object
+    description: |
+      Authorization section for an event type. This section defines three access control lists: one for producing
+      events ('write'), one for consuming events ('read'), and one for administering an event type ('admin').
+      Regardless of the values of the authorization properties, administrator accounts will always be authorized.
+    properties:
+      admin:
+        type: array
+        description: |
+          An array of subject attributes that are required for updating the event type. Any one of the attributes
+          defined in this array is sufficient to be authorized. The wildcard item takes precedence over all others,
```

Comment:
I have, but I think we discussed this in alignment, and decided that it would be too easy for a user to accidentally open their event type to all. Having to explicitly include a wildcard makes such errors less likely.